docs/user/ppl/functions/datetime.rst

```diff
@@ -1152,6 +1152,81 @@ Example::
     +--------------------------+-----------------------------+
 
 
+UTC_DATE
+-------
+
+Description
+>>>>>>>>>>>
+
+Returns the current UTC date as a value in 'yyyy-MM-dd'.
```

Comment:
Documentation above says: YYYY-MM-DD
Is it better to be consistent?

Reply:
Made consistent with MySQL. Addressed in https://github.com/Bit-Quill/opensearch-project-sql/commit/59ef32f37ea15e9f44cae06574c3cb9f27db26e1.